skrf/circuit.py

```diff
@@ -137,6 +141,7 @@ def networks_dict(self, connections=None, min_nports=1):
                 ntws.append(ntw)
         return {ntw.name: ntw for ntw in ntws  if ntw.nports >= min_nports}
 
+    @property
     def networks_list(self, connections=None, min_nports=1):
```

Comment:
properties dont take arguments,

Reply:
Ho indeed, you're right. It's corrected now.